### PR TITLE
Glossary

### PR DESCRIPTION
**Pull request by mcherem**
Glossary draft 2 per DOCS-3989 - please review

### CODE REVIEW

**Review by mcherem — COMMENTED** [review]
Please review and comment.

File: modules/ROOT/pages/glossary.adoc [hunk 1]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
-
-B2B transaction-processing solution:: Functionality that serves as a translation layer between internal systems and the various formats (X12, EDIFACT, XML, CSV, and others) and transport protocols (AS2, FTP/S, and others) commonly used in electronic transactions between companies.
+Activity:: Role-based document transaction tracking (for send and receive) based on rules, actions, and functions.
 
-B2B Transaction-Processing Framework:: B2B transaction-processing solution that enables developers to build a single set of xref:mule-runtime::index.adoc[Mule Runtime] flows that can process a wide range of messages from many different partners by dynamically applying rules and configuration data stored in xref:anypoint-partner-manager.adoc[Anypoint Partner Manager] (APM)
+Anypoint Partner Manager:: A single web-based user interface for Partner Configuration, Document flow  and Transaction Monitoring. The functionality that serves as a translation layer between internal systems and the various formats (X12, EDIFACT, XML, CSV, and others) and transport protocols (AS2, FTP/S, and others) commonly used in electronic transactions between companies.
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:20]: Needs a comma after "Document flow". Should also put "Document" and "Transaction Monitoring" in lowercase. However, if Document and Transaction Monitoring are defined elsewhere, please embed a cross reference link to the respective entry, or at the least, cast cross reference words in italics.

File: modules/ROOT/pages/glossary.adoc [hunk 2]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
 
-[[sectd]]
-*D*
-
-Document:: An instance of a structured payload data being passed through the B2B system.  Synonymous with other terms commonly used in various integration-related domains such as file, interchange, payload, message, and transaction.
-
-Document Definition:: A specific document format that is configured in the system.  A Document Definition is identified by the following pieces of information: Partner, Standard, Version, and Message Type. Schemas, Maps, and Document Properties are associated with a Document Definition.
-
-Document Properties:: Document Properties are data fields that are extracted from received documents for the purpose of correlation, duplicate checking, and searching in the B2B Transactions screen.  Document Properties are configured for Document Definitions.  The user provides the Name of the property and the path (XPath for XML, JSONPath for others) for retrieving the property from the document.  These properties are extracted and stored by the Tracking service.
+Activity API:: Provides users with the ability to obtain transaction status???
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:22]: "???" -- Would be better to leave out questions to reviewers from doc that may be seen by others.

File: modules/ROOT/pages/glossary.adoc [hunk 3]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
 
 [[secte]]
 *E*
 
-Environment ID:: String that uniquely identifies an xref:access-management::environments.adoc[environment].
+Endpoint:: Defines the protocol, address, and other details specific to an exchange of messages between partners such as Acknowledgement, Source Endpoint, and Target Endpoint.
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:27]: This definition doesn't make sense to me. I think of an endpoint as a URL to a server. I don't understand how an endpoint can define a protocol, address, etc. Also not sure what you mean by "Acknowledgement, Source Endpoint, and Target Endpoint"

File: modules/ROOT/pages/glossary.adoc [hunk 4]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
 
-External Partner:: Any company registered in APM that is not the Home Partner.
+[[sectg]]
+*G*
+
+Global Validation and Acknowledgement settings:: Default Host settings that apply to all partners. More here???
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:32]: Acknowledgement is a British spelling. In the US, it's Acknowledgment. Please fix globally.
Again, better to not put reviewer questions in an external doc.

File: modules/ROOT/pages/glossary.adoc [hunk 5]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
 
 [[secth]]
 *H*
 
-Home Partner:: The company that is subscribing to Partner Manager, considered the “internal” partner.
+Host:: The company that is subscribing to the Host in Partner Manager.
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:37]: This is unclear. Would be better to provide more detail to explain the two sides of the communication. I don't understand how a host would subscribe to a Partner Manager Host.

It's also better to use "organization" instead of "company" as not everyone using B2B is in a company. For example, a non-profit, or the government.

File: modules/ROOT/pages/glossary.adoc [hunk 6]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
+*I*
 
-Lookup Table:: A table that facilitates transaction processing by associating values.
+Identifier:: Uniquely identifies a particular Partner. Identifiers have a Type and Qualifier that indicate the identifier scheme and Value.
 
 [[sectm]]
 *M*
 
-Map:: Stores map type (XSLT or DataWeave) and script for transforming from one Document Definition to another Document Definition.
+Map:: Stores map type (XSLT or DataWeave) and scripting for transforming from one Message to another.
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:47]: Should be "Stores a map type". Should put Message in lowercase.

File: modules/ROOT/pages/glossary.adoc [hunk 7]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
-Partner:: A company that exchanges B2B messages.
-
-Partner Identifier:: A predefined identifier of a particular Partner.  Partner Identifiers have Type and Qualifier that indicate the identifier scheme and Value.
+Partner Manager API:: Manages partner-related configurations such as document definitions and partner profiles.
 
 [[sectr]]
 *R*
 
-Route:: Connects a source Channel to a Target Channel.
+Retry:: A redo of a message type that has previously failed.
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:62]: Should explain what the message types are or provide an example. "redo" is also vague in that the action is a mystery--is it resubmitting, or requerying, or what?

File: modules/ROOT/pages/glossary.adoc [hunk 8]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
 
-Schema:: The schema is stored with the Document Definition and is one of the following depending on the message type:
-
-* XML: XSD file
-* CSV: (NA)
-* EDI: EDI Schema Language (ESL) file
-
-
-Standard APM Field Format:: Describes fields for which valid values can include letters, numbers, or the dash ( *-* ) and underscore ( *_* ) characters.
+Source Endpoint:: The origin of the message flow that includes an endpoint and a message type. Hosted by the Host Partner and are often defined by a Host API.???
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:67]: Why did you remove "Schema"? That seems like an important concept.

File: modules/ROOT/pages/glossary.adoc [hunk 9]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
 
 [[sectt]]
 *T*
 
-Transaction:: The end-to-end processing of a single instance of a document from receipt at an endpoint all the way to the desired Business Service.  A TransactionId can be used to find metadata and persisted payloads for the transaction at each stage.
+Target Endpoint:: Final destination endpoint. Hosted by the Host Partner and are often defined by a Host API. Depending on the direction, this is either an endpoint for sending to a partner or a target at the host if receiving from a partner.???
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:72]: "are often" should be "is often". Please define what is being sent to a partner or receiving from a partner.

File: modules/ROOT/pages/glossary.adoc [hunk 10]
```diff
@@ -15,81 +15,69 @@ endif::[]
...
+
+Trading Partner:: Any external company with which your company does business and is registered in Anypoint Partner Manager that is not the Host Partner. Your company itself is also considered a Trading Partner. Must have at least a name and identifier associated with it.
+
+Transaction:: End-to-end aspect between target and source is the document flow.
 
-Transaction Definition:: The full configuration that configures how a document is processed between two partners.
+Transmission:: An event in which a bundle of messages are sent between partner and host. For example, map or transmission to a B2B payload.
```
Comment (george-hoffer) [inline modules/ROOT/pages/glossary.adoc:78]: "map" should be "a map"
